### ISSUE TITLE
About Minecraft 1.13

### ISSUE THREAD

**Issue by FlyingPikachu** [neolumia/vault#5]
Hello @conreptio ,

Thank you for hosting what is likely the most popular fork of Vault on Spigot. Seeing as you decided to include the VaultAPI in the project a month ago, I'm curious as to whether or not you intend to update it for Minecraft 1.13.

Currently, VaultAPI uses subtype IDs for some methods, which if I'm not mistaken will be deprecated in 1.13. Also, item info for all the new blocks must be added (and perhaps for some that are missing like Material.WOODEN_DOOR). There are also bugs to be addressed, like [the one I fixed on the main repo last week](https://github.com/MilkBowl/VaultAPI/commits/master)).

In short, Quests and a number of other plugins rely on this API but are faced with the likelihood it won't be updated. Do you have any plans to the contrary?
